firrtl/src/test/scala/firrtlTests/SerializerSpec.scala

```diff
@@ -179,6 +179,45 @@ class SerializerSpec extends AnyFlatSpec with Matchers {
     SMemTestCircuit.circuit(ReadUnderWrite.Old).serialize should include("old")
   }
 
+  it should "support emitting Probe/RWProbe types and related expressions/statements" in {
+    val probeInt = DefWire(NoInfo, "foo", ProbeType(UIntType(IntWidth(3))))
+    Serializer.serialize(probeInt) should be("wire foo : Probe<UInt<3>>")
+
+    val rwProbeBundle = Port(
+      NoInfo,
+      "foo",
+      Output,
+      RWProbeType(BundleType(Seq(Field("bar", Default, UIntType(IntWidth(32))))))
+    )
+    Serializer.serialize(rwProbeBundle) should be("output foo : RWProbe<{ bar : UInt<32>}>")
+
+    val probeVec = Port(
+      NoInfo,
+      "foo",
+      Output,
+      RWProbeType(VectorType(UIntType(IntWidth(32)), 4))
+    )
+    Serializer.serialize(probeVec) should be("output foo : RWProbe<UInt<32>[4]>")
```

Comment:
I'd expect this to be `RWProbe<UInt<32>[4]>`?

Reply:
I think what you typed is the same as what I have?

Reply:
*blinks a few times* ... Wow, yes you're right.  I remember seeing this as `RWProbe<UInt<32>>[4]`, but don't know where that came from.  Whoops!